conformance/results/kong-kubernetes-ingress-controller.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
+date: "2023-07-17T14:54:07+02:00"
+gatewayAPIVersion: v0.7.1
+implementation:
+  organization: kong
+  project: kubernetes-ingress-controller
+  url: github.com/kong/kubernetes-ingress-controller
+  version: "2.10.3"
+  contact:
+    - '@czeslavo'
+    - '@mflendrich'
+    - '@mlavacca'
+    - '@pmalek'
+    - '@rainest'
+    - '@programmer04'
+    - '@randmonkey'
```

Comment:
Were we going to put a Github issue link in here instead?

Reply:
Yeah, I've updated it and created a PR to highlight this possibility in the GEP and the godoc: https://github.com/kubernetes-sigs/gateway-api/pull/2234